types/externs.d.ts

```diff
@@ -176,6 +185,8 @@ declare global {
       throttlingMethod?: 'devtools'|'simulate'|'provided';
       /** The throttling config settings. */
       throttling?: ThrottlingSettings;
+      /* Optional object of user-facing strings to be used instead of Lighthouse's given values. */
+      displayStrings?: DisplayStrings;
```

Comment:
seeing it now, lhr.configSettings.displayStrings` isn't the clearest that it's not the equivalent of `rendererFormattedStrings`, its _user_ defined displayStrings, but I'm also ok with it if you like the name :)